css/media.css

```diff
@@ -0,0 +1,35 @@
+@media screen and (max-width: 768px) {
+  .header_container {
+    flex-direction: column;
+    text-align: center;
```

Comment:
your media query doesn't work, it looks like this 

![Screenshot from 2021-03-03 11-21-01](https://user-images.githubusercontent.com/47992412/109783415-a5ed9b00-7c12-11eb-936e-25cb3b9c20b2.png)